src/lib/DepositRecordSerializer.test.js

```diff
@@ -0,0 +1,171 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { DepositRecordSerializer } from "./DepositRecordSerializer";
+
+
+describe('Record serializer', () => {
+  describe('removeEmptyValues', () => {
+    const serializer = new DepositRecordSerializer();
+    const record = {
+      contributors: [
+        { identifiers: [] },
+      ],
+      version: 0,
+      cool: false,
+      creators: [
+        null,
+        undefined,
+        {},
+      ],
+      description: ""
+    }
+
+    const cleanedRecord = serializer.removeEmptyValues(record);
+
+    expect(cleanedRecord).toEqual({version: 0});
```

Comment:
Shouldn't it return `{ cool: false, version: 0}`?

Reply:
The logic of `removeEmptyValues` wasn't altered, so you tell me :) . But seriously, I think you are right actually: if the `missing` value for that Boolean field on the backend is True, then when it is set to False on the frontend, the serialization would strip it and the backend not seeing it would set the field back to True... This is actually true for all fields... :worried: 

Let's discuss how we want to proceed on that one.

Reply:
@fenekku can you create an issue for that? Then, you can go ahead and merge this/release :)

Reply:
:+1: Created: https://github.com/inveniosoftware/react-invenio-deposit/issues/43